apis/nucleus/src/components/Cell.jsx

```diff
@@ -471,6 +470,12 @@ const Cell = forwardRef(
         getQae() {
           return state.sn.generator.qae;
         },
+        getExtDefinition() {
```

Comment:
It says "getExtensionDefinition" in the "--- Viz api ---" comment below...

Reply:
yea this is inside the cell, haven't added it to the Viz one yet, going to hold off on it a bit as it requires us to resolve the property panel definition. Its not strictly needed yet either.